LogMonitor/src/LogMonitor/sample-config-files/IIS/LogMonitorConfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "LogConfig": {
+	  "logFormat": "json",

```

Comment:
nit: remember to use `spaces` here instead of `tab`; can fix that setting on your VS. It's mis-aligned with the rest as you can see on L4+

Reply:
Addressed